web/build.sh

```diff
@@ -113,3 +114,15 @@ if builder_start_action test; then
 
   builder_finish_action success test
 fi
+
+coverage_action() {
+  builder_echo "Creating coverage report..."
+  pushd .. > /dev/null
```

Comment:
Could just do `cd ..` and `cd web` at the end as we know what the folders are, but for clarity I suggest we do:

```suggestion
  cd "$KEYMAN_ROOT"
```